src/code/z_view.c

```diff
@@ -275,7 +275,7 @@ s32 View_StepDistortion(View* view, Mtx* projectionMtx) {
 /**
  * Apply view to POLY_OPA_DISP, POLY_XLU_DISP (and OVERLAY_DISP if ortho)
  */
-void View_Apply(View* view, s32 mask) {
+BAD_RETURN(s32) View_Apply(View* view, s32 mask) {
```

Comment:
Just checked, this matches both debug and retail:
```c
s32 View_Apply(View* view, s32 mask) {
    mask = (view->flags & mask) | (mask >> 4);

    if (mask & VIEW_PROJECTION_ORTHO) {
        return View_ApplyOrtho(view);
    } else {
        return View_ApplyPerspective(view);
    }
}
```

Reply:
ooo those functions return things, ok thats good